### ISSUE TITLE
Usability issue: Position of the ruleset icon (puzzle icon)

### ISSUE THREAD

**Issue by Gyokur0** [gorhill/uMatrix#983]
The lock, the eraser and the reload button are normally used in conjunction. The mouse distance between them are now longer too.
I would prefer for those three to stand together. 

Therefore I would move the new puzzle icon left of the lock icon.

(Question on the side: How are the new ruleset definitions are delivered to the addon? I hope not that uMatrix does a request on every page I visit to see if there are some rulesets to download? )

**Comment by Thorin-Oakenpants** [comment]
> How are the new ruleset definitions are delivered to the addon

Dashboard>Assets> check the ruleset recipes, it's stored locally and updated like other assets

**Comment by Gyokur0** [comment]
@Thorin-Oakenpants thanks for the info.

**Comment by gorhill** [comment]
The puzzle icon is a scope-related one, all scope-related icons appears near the scope selector, so I do not intend to change this.

**Comment by Gyokur0** [comment]
Hi gorhill!
Perhaps I described it too complicated, of course it should stay near the scope selector, but the actions for save, revert and reload should be more together. 

In the popup.html the line with id=buttonRecipes should be simply be moved up two lines between buttonMtxSwitches and buttonPersist.






![mockup](https://user-images.githubusercontent.com/29225198/37982838-f78577da-31f1-11e8-93b2-bdf3695c69c8.png)

**Comment by gorhill** [comment]
Ok, I agree it makes sense.

**Comment by Thorin-Oakenpants** [comment]
I'd even go slightly further. The puzzle-piece is optional and doesn't always show, so you don't want the items to the right sliding left (muscle memory etc). Or maybe that's just me and some weird sense of strict order kicking in

![37982838-f78577da-31f1-11e8-93b2-bdf3695c69c8](https://user-images.githubusercontent.com/16656956/37985250-cca0761e-31e7-11e8-8921-d04368f9ba21.png)

Above left group is all `scope setting`, the right hand group of three tie in with the global revert, but maybe you want a gap, maybe a centered middle div. Or maybe the group of three belong to the left since they are for the current scope

if any that makes sense, food for thought, that's all

**Comment by Gyokur0** [comment]
That makes totally sense, this is even better.

The icons on the left are used seldom or should with thought, e.g. you'd be sure what you are doing if you change the scope to the wildcard star, or even disable uMatrix completely, the other two are used more rarely.

The stuff on the right is used far more often. 

From my perspective I have strict base rules, so on every new unknown site, I have to enable and reload (more than once if a lot of external content is there). If I like the site I hit the lock. More rarely I hit the eraser, often in experimenting with video in pages. I never have used the revert all button (I just closed the browser)

I'm not sure if putting the reload button directly next to the revert all is a good idea (which is a destructive action, that also affects more than one page). 

I tried to optimize this in version 1 in the mockup.

In the second version I switched lock and erase since I use lock and reload more often together. (But I'm also aware undoing an accidental lock click is more complicated that undoing a eraser click)

![mockup2](https://user-images.githubusercontent.com/29225198/38010365-fd38d426-3258-11e8-9d98-23e0a619c60a.png)


This is based on my experience and usage and I certainly cannot speak for everyone.
The only thing I want is that the gap between reload and lock is not so large, so every presented solution is better than the status quo ;)
